### PR TITLE
rename new-people to people

### PR DESCRIPTION
**Pull request by tkellen**
this moves your new version of people into production!

### CODE REVIEW

File: deploy/ansible/inventory/staging [hunk 1]
```diff
@@ -3,4 +3,4 @@
 # production server's FQDN. This is likely the only change you will need to
 # make between production and staging.
 
-new-people-staging.bocoup.com site_fqdn=new-people-staging.bocoup.com commit=master
+people-staging.bocoup.com site_fqdn=people-staging.bocoup.com commit=master
```
Comment (leobalter) [inline deploy/ansible/inventory/staging]: for later improvement, these URLs should be `staging-__name__.bocoup.com` so it's just about prefixing the whole url

**Comment by leobalter** [comment]
LGTM, we need to address these links on bocoup/infrastructure as well.

**Comment by tkellen** [comment]
handled here:

**Comment by tkellen** [comment]
handled here:
https://github.com/bocoup/infrastructure/commit/505c1491e1e2fd9671de03db31a887b1bf838da8

**Pull request merged**